src/kirby/components/icon/icon.component.scss

```
@@ -1,5 +1,30 @@
+@import '../../scss/web-imports';
+
 :host {
-	display: flex;
-    align-items: center;
-    justify-content: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  
+  ion-icon {
+    font-size: size('m');
+  }
+
+  &.sm {
+    ion-icon {
```

Consider flattening:
```suggestion
    :host(.sm) ion-icon {
```